packages/plugins/ethereum-wallet/Cargo.toml

```diff
@@ -8,19 +8,15 @@ rust-version.workspace = true
 license.workspace = true
 
 [dependencies]
-polywrap_core.workspace = true
 polywrap_plugin.workspace = true
-polywrap_msgpack_serde.workspace = true
-polywrap_resolvers.workspace = true
-wrap_manifest_schemas.workspace = true
 serde.workspace = true
 thiserror.workspace = true
-serde_bytes.workspace = true
+
 tokio.workspace = true
 lazy_static.workspace = true
 
 ethers = { version = "2.0.6" }
 
-
 [dev-dependencies]
 polywrap_client.workspace = true
+polywrap_resolvers.workspace = true
```

Comment:
Doesn't the polywrap_client already re-export resolvers?

Reply:
true! nice catch